### DISCUSSION TITLE
How do I use `quarkus.rest-client.providers` in REST Client Config application.yml?

### DISCUSSION THREAD

**Discussion by vincejv** [quarkusio/quarkus#28923] [Q&A]
In `application.yml` I tried the following and doesn't work as the logs show this
```
Unrecognized configuration key "quarkus.rest-client.providers."com.example.provider"" was provided; it will be ignored
```
```yaml
quarkus:
  rest-client:
    providers:
      com.example.provider: 1
```

but in Quarkus documentation, it describes this property as

> Map where keys are fully-qualified provider classnames to include in the client, and values are their integer priorities. The equivalent of the @RegisterProvider annotation. Can be overwritten by client-specific settings.

![image](https://user-images.githubusercontent.com/1276544/198853573-8b1120a8-8e2a-4b99-ba9a-3d42f584bbae.png)

[a comment by an automated account is not shown]

**Reply by Sgitario** [reply]
The right property should be:

```
quarkus:
  rest-client:
    providers: com.example.provider
```

Have you tried it?
